Source/Scene/Model/MetadataPipelineStage.js

```diff
@@ -62,60 +83,225 @@ MetadataPipelineStage.process = function (
   frameState
 ) {
   const { shaderBuilder, model } = renderResources;
+  const { structuralMetadata = {}, content } = model;
```

Comment:
If `model.structuralMetadata` is defined, the empty object will not be allocated. [According to MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#default_value),

> The default value can be any expression. It will only be evaluated when necessary.